spec/models/allocation_spec.rb

```diff
@@ -150,7 +162,13 @@
       expect(allocation.moves.first.cancellation_reason).to eq(Move::CANCELLATION_REASON_OTHER)
```

Comment:
I need to fix this - it should be testing the move `cancellation_reason_comment`. Also need to agree the behaviour if no cancellation reason comment is passed to the new endpoint - this is an optional field so should we just allow and persist blank comments?